subiquity/controllers/filesystem.py

```diff
@@ -399,12 +399,15 @@ def delete_mount(self, mount):
 
     def create_filesystem(self, volume, spec):
         if spec['fstype'] is None:
+            if getattr(volume, 'flag', None) != 'prep':
+                volume.wipe = None
```

Comment:
we *always* wipe prep, IIRC;  leave a comment here?

Reply:
yeah good idea